ws-nextjs-app/cypress.config.ts

```diff
@@ -47,6 +47,12 @@ export default defineConfig({
         baseUrl: 'http://localhost:7081',
       },
     },
+    testIsolation: false,
   },
+  defaultCommandTimeout: 10000,
+  pageLoadTimeout: 60000,
```

Comment:
Is there a reason we're not making the pageLoadTimeout `100000` to match the express app?

Reply:
Not particularly, just thought 100secs seemed a little excessive to wait for a timeout, but maybe there was historical reasons why 🤷